crates/sound/src/vhu_sound.rs

```diff
@@ -381,7 +381,36 @@ impl VhostUserSoundThread {
                         .map_err(|_| Error::DescriptorWriteFailed)?;
                     len = desc_response.len();
                 }
-                VIRTIO_SND_R_PCM_START | VIRTIO_SND_R_PCM_STOP | VIRTIO_SND_R_PCM_RELEASE => {
+                VIRTIO_SND_R_PCM_RELEASE => {
+                    let pcm_hdr = desc_chain
+                        .memory()
+                        .read_obj::<VirtioSoundPcmHeader>(desc_request.addr())
+                        .map_err(|_| Error::DescriptorReadFailed)?;
+                    let stream_id = u32::from(pcm_hdr.stream_id);
+                    let state = audio_backend.get_stream_state(stream_id).unwrap();
```

Comment:
Should we propagate a `get_stream_state()` error?

Reply:
Or handle it in some way, like responding with `VIRTIO_SND_S_IO_ERR`?

Reply:
I prefer the later

Reply:
I did the former ;)